### PR TITLE
Plug search improvements

### PR DESCRIPTION
**Pull request by peteruithoven**
Alterations to get better results when searching for sub settings. See: https://github.com/elementary/applications-menu/pull/133
screenshot from 2018-10-03 22 25 12
screenshot from 2018-10-03 22 24 54

Not sure about that double defaults result... I could try filtering those out, but It can enable more search terms.

### CODE REVIEW

File: src/Plug.vala [hunk 1]
```diff
@@ -96,13 +96,14 @@ public class ApplicationsPlug : Switchboard.Plug {
...
         search_results.set ("%s → %s".printf (display_name, _("Default Apps")), DEFAULTS);
         search_results.set ("%s → %s".printf (display_name, _("Default Application")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Web Browser")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Music Player")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Email Client")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Text Editor")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default File Browser")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Video Player")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Calendar")), DEFAULTS);
+        search_results.set ("%s → %s → %s".printf (display_name, _("Default"), _("Web Browser")), DEFAULTS);
```
Comment (danrabbit) [inline src/Plug.vala]: While we're changing these, can we put them in alphabetical order?

Comment (peteruithoven) [reply src/Plug.vala]: I could, but my logic was following the order they appear in the UI, that makes it a bit easier to compare them. But I can totally change it to be alphabetically.

Comment (danrabbit) [reply src/Plug.vala]: Ah yeah. I think we probably need to redesign this page in the UI. It seems like a pretty random ordering. I feel like alphabetical would make it easier to find a specific line regardless of what the UI side looks like, but I can see wanting to try to mirror the UI as well. In that case, I think I'm fine with whichever

**Comment by danrabbit** [comment]
Gonna mark this as "blocked" until we get a stable Juno release

**Pull request merged**